test/Recipe-test.js

```diff
@@ -0,0 +1,95 @@
+import { expect } from "chai";
+import ingredientsData from "../src/data/ingredients";
+import recipeSampleData from "../src/sampleSets/recipeSample";
+import Recipe from "../src/classes/Recipe";
+
+describe("Recipe", function () {
+  let recipe1, recipe2, recipe3;
```

Comment:
I like this never heard or seen anything like this... interesting way to declare variables.